src/core/annotation.js

```diff
@@ -4070,6 +4070,20 @@ class InkAnnotation extends MarkupAnnotation {
     ink.set("Border", [0, 0, 0]);
```

Comment:
I don't think this line makes sense now that you set a `BS`-entry below, since in our own code we ignore such an entry in that case; note https://github.com/mozilla/pdf.js/blob/071e6bc7e77754d9f1f13a84a05851d23a657971/src/core/annotation.js#L797-L835